src/cmd_argument.rs

```diff
@@ -61,6 +61,36 @@ impl CmdArgument for Vec<String> {
     }
 }
 
+/// Similar to the implementation for [`Vec<&str>`].
+/// All elements of the array will be passed into the child
+/// process as arguments, **without** splitting them by whitespace.
+///
+/// ```
+/// use stir::cmd;
+///
+/// let output: String = cmd!(&["echo", "foo"]);
+/// assert_eq!(output, "foo\n");
+/// ```
+#[rustversion::since(1.51)]
+impl<const N: usize> CmdArgument for &[&str; N] {
```

Comment:
Should this be `for [&str; N]`, to avoid the `&`?

Reply:
Oh, right. I didn't realize this was possible. Then the `cmd!(["echo", "foo"])` syntax actually will work.

Do we then want another impl for `&[&str]`? I guess that can also wait for another PR...